src/components/Thought.js

```diff
@@ -92,11 +93,12 @@ export const Thought = connect(({ cursor, cursorBeforeEdit, expanded, expandedCo
     canDrag: props => {
       const thoughtMeta = meta(pathToContext(props.thoughtsRankedLive))
       const contextMeta = meta(contextOf(pathToContext(props.thoughtsRankedLive)))
+      const sortPreference = getSortPreference(contextMeta)
       return (!isMobile || globals.touched) &&
         !thoughtMeta.immovable &&
         !thoughtMeta.readonly &&
         !(contextMeta.readonly && contextMeta.readonly.Subthoughts) &&
-        !(contextMeta.immovable && contextMeta.immovable.Subthoughts)
+        !(contextMeta.immovable && contextMeta.immovable.Subthoughts) && sortPreference !== 'Alphabetical'
```

Comment:
Hmmm unfortunately this prevents dragging a thought into a sibling. Perhaps you can make this check during `canDrop`.